qemu/tests/seabios_scsi_lun.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    [seabios] Verify guest works well when using virtio-scsi with LUN not 0
+
+    this case will:
+    1) Boot guest with virtio-scsi disk and lun is not 0.
+    2) Check boot menu list.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_output(session_obj):
+        """
+        Use the function to short the lines in the scripts
+        """
+        return session_obj.get_stripped_output()
+
+    def boot_menu():
+        return re.search(boot_menu_hint, get_output(seabios_session))
+
+    def get_list():
+        return re.findall(r"^\d+\. (.*)\s", get_output(seabios_session), re.M)
+
+    timeout = float(params.get("login_timeout", 240))
```

Comment:
This test doesn't perform a login into the guest os so the param name shouldn't be `login_timeout`, the name `boot_timeout` would be fine, also, `240` is too large for this test, I think `60` is okay.

Reply:
Updated, thanks!